src/main/kotlin/net/axay/blaubot/commands/Admin.kt

```diff
@@ -25,6 +32,16 @@ object Admin : SlashCommand(
         interaction.command.subCommands["chat"]?.options?.get("message")?.string()?.let {
             interaction.channel.createMessage(it)
         }
+        
+        interaction.command.subCommands["embed"]?.options?.get("message")?.string()?.let {
+            interaction.acknowledge(false).followUp {
+                embed {
+                    title = interaction.command.subCommands["embed"]?.options?.get("title")?.string()
+                    image = interaction.member.asUser().avatar.url
+                    description = it
```

Comment:
`description = it` seems to be wrong, as `it` is the embed builder in this case, not the description.

Reply:
No, according to IntelliJ, `it` is the string and` this` is the EmbedBuilder

Reply:
Ah yes you are right.
But please only get the embed command options once and save them inside of a variable.
`val embedOptions = interaction.command.subCommands["embed"]?.options`

Reply:
Then you can do `embedOptions?.get("title")` and `embedOptions?.get("message")`

Reply:
Did it